IBPSA/Fluid/Actuators/Dampers/PressureIndependent.mo

```diff
@@ -3,15 +3,39 @@ model PressureIndependent
   "Model for an air damper whose mass flow is proportional to the input signal"
   extends IBPSA.Fluid.Actuators.BaseClasses.PartialDamperExponential(
     final linearized=false,
+    final casePreInd=true,
     from_dp=true);
-  input Real phi = l + y_actual*(1 - l)
+  input Real phi = l + y_internal*(1 - l)
     "Ratio actual to nominal mass flow rate of damper, phi=kDam(y)/kDam(y=1)";
   parameter Real l2(unit="1", min=1e-10) = 0.01
     "Gain for mass flow increase if pressure is above nominal pressure"
     annotation(Dialog(tab="Advanced"));
   parameter Real deltax(unit="1", min=1E-5) = 0.02 "Transition interval for flow rate"
     annotation(Dialog(tab="Advanced"));
 protected
+  parameter Real y_min = 2E-2
+    "Minimum value of control signal before zeroing of the opening";
+  parameter Integer sizeSupSplBnd = 5
+    "Number of support points on each quadratic domain for spline interpolation";
+  parameter Integer sizeSupSpl = 2 * sizeSupSplBnd + 3
+    "Total number of support points for spline interpolation";
+  parameter Real[sizeSupSpl] ySupSpl_raw = cat(
+    1,
+    linspace(1, yU, sizeSupSplBnd),
+    {yU-1/3*(yU-yL), (yU+yL)/2, yU-2/3*(yU-yL)},
```

Comment:
can we document that rationale for this implementation somewhere?

Reply:
Addressed